InvenTree/InvenTree/version.py

```diff
@@ -23,6 +25,53 @@ def inventreeVersion():
     return INVENTREE_SW_VERSION
 
 
+def inventreeVersionTuple(version=None):
+    """ Return the InvenTree version string as (maj, min, sub) tuple """
+
+    if version is None:
+        version = INVENTREE_SW_VERSION
+
+    match = re.match(r"^.*(\d+)\.(\d+)\.(\d+).*$", str(version))
+
+    return [int(g) for g in match.groups()]
+
+
+def versionTupleToInt(version):
```

Comment:
You can compare version tuples directly.  

(2,4,1) < (2,4,2):  True
(3) > (2,4,2): True

Reply:
Huh, you learn something new every day. I did not know Python could do that!